test/apps/iodemo/io_demo.cc

```diff
@@ -694,7 +707,9 @@ class P2pDemoCommon : public UcxContext {
                                          test_opts.chunk_size),
                            "data iovs"),
         _data_chunks_pool(test_opts.chunk_size, "data chunks",
-                          test_opts.memory_type, test_opts.num_offcache_buffers)
+                          test_opts.memory_type, test_opts.num_offcache_buffers),
+        _zcopy_chunks_pool(test_opts.chunk_size, "zcopy chunks",
+                          UCS_MEMORY_TYPE_UNKNOWN, test_opts.num_offcache_buffers)
```

Comment:
why need separate pool of `chunk_size` for saving pointers to data?

Reply:
different type Buffer needs different pool. Since the idea was to support a new Buffer type, we need another pool for that.